awx/main/tests/functional/conftest.py

```diff
@@ -901,3 +920,8 @@ def control_plane_execution_environment():
 @pytest.fixture
 def default_job_execution_environment():
     return ExecutionEnvironment.objects.create(name="Default Job EE", managed=False)
+
+
+@pytest.fixture
+def fixture_compute_cloud_inventory_sources():
```

Comment:
No need to use “fixture” in fixture names. It's common to use the produced thing name, as that's what's injected into the tests. Additionally, fixtures should have docstrings for `pytest --collect-only --fixtures`.
```suggestion
def computed_cloud_inventory_sources() -> dict[str, str]:
    """A mapping of cloud inventory source names to labels."""
```

Reply:
I ended up not needing this so it has been deleted.